src/utils.rs

```diff
@@ -0,0 +1,11 @@
+// #[allow(dead_code)]
```

Comment:
This was the default. Why the removal and why a comment instead of delete?

Reply:
Oops I meant to uncomment this. I'll do that now. Thanks.